securitas/themes/default/templates/main.html

```diff
@@ -0,0 +1,54 @@
+{% extends "master.html" %}
+
+
+{% block head %}
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link href="{{ url_for('theme.static', filename='css/default.css') }}" rel="stylesheet" type="text/css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+{% endblock %}
+
+{% block content %}
+    <nav class="navbar navbar-expand-lg navbar-dark bg-info justify-content-between mb-5 border-bottom">
+        <a class="navbar-brand" href="/">sēcūritās</a>
+        {% if current_user %}
+        <form><input id="search" type="search" class="typeahead form-control" autocomplete="off" placeholder="search..."></form>
+        <ul class="navbar-nav">
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    {{ current_user.username }}
+                </a>
+                <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                    <a class="dropdown-item" href="{{ url_for('user', username=current_user.username) }}">Profile</a>
+                    <a class="dropdown-item" href="{{ url_for('logout') }}">Log Out</a>
+                </div>
+            </li>
+        </ul>
+        {% endif %}
+    </nav>
+    {% with flashes = get_flashed_messages(with_categories=True) %}
+        {% if flashes %}
+        <div class="container flash-messages">
+            {% for category, flash in flashes %}
+            <div class="card-panel {{ category }} lighten-4 {{ category }}-text text-darken-4">
+                {{ flash|striptags }}
+            </div>
+            {% endfor %}
+        </div>
+        {% endif %}
+    {% endwith %}
+{% endblock %}
+
+{% block footer %}
+    <footer class="bg-light border-top py-5 mt-4">
+        <div class="container h6 text-center">
+        <p>&copy; 2020. Powered by <a href="https://github.com/fedora-infra/securitas">securitas</a></p>
```

Comment:
- We can probably just drop the copyright, as there's implicit copyright, at least in the US and I assume similar elsewhere. Otherwise, we probably want to add `Red Hat, Inc.` after the `&copy;`.
- `.` after `</a>`